Marlin/Configuration.h

```diff
@@ -892,6 +892,11 @@ const bool Z_MIN_PROBE_ENDSTOP_INVERTING = false; // set to true to invert the l
 //
 //#define PANEL_ONE
 
+
+// Cartesio UI
+// http://mauk.cc/webshop/cartesio-shop/electronics/user-interface
+//#define CartesioUI
```

Comment:
Our convention is to put these settings in all-caps. So this should be something like `CARTESIOUI` or `CARTESIOUI_CONTROLLER`.